src/ResourceManager/Network/Commands.Network/help/Set-AzureRmNetworkSecurityGroup.md

```diff
@@ -37,7 +37,7 @@ The command persists the modified Azure network security group using **Set-Azure
 Run cmdlet in the background
 
 ```yaml
-Type: System.Management.Automation.SwitchParameter
+Type: SwitchParameter
```

Comment:
You'll need to regenerate the help files.

Remember to use the flag `-UseFullTypeName` when you generate the help files. instructions [here](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-a-single-markdown-file).